cdap-master/src/test/java/io/cdap/cdap/master/environment/plugin/ConstantCallable.java

```diff
@@ -19,16 +19,16 @@
 import io.cdap.cdap.api.annotation.Name;
 import io.cdap.cdap.api.annotation.Plugin;
 import io.cdap.cdap.api.plugin.PluginConfig;
-import io.cdap.cdap.master.environment.ServiceWithPluginApp;
 
 import java.util.concurrent.Callable;
 
 /**
  * Used to test plugins in apps.
  */
-@Plugin(type = ServiceWithPluginApp.PLUGIN_TYPE)
+@Plugin(type = ConstantCallable.PLUGIN_TYPE)
 @Name(ConstantCallable.NAME)
 public class ConstantCallable implements Callable<String> {
+  public static final String PLUGIN_TYPE = "callable";
```

Comment:
the way plugins are being used in the app, it will accept any plugin that implements `Callable<String>`, so I think it's actually better in the app.

Reply:
I see. Make sense. Drop the PR.